### PR TITLE
Feature/ProductVariant&RegularCustomer

### LINKED ISSUE #70: Add Bar code and purchasing and sale price in variant of a product

### LINKED ISSUE #74: Description field for Regular customer

### PR DESCRIPTION
**Pull request by ammarhaiderak**
closes #74 #70

### CODE REVIEW

File: src/components/sales/AddEditProduct.vue [hunk 1]
```diff
@@ -65,16 +65,18 @@
...
-              <col span="1" style="width: 12%;">
-              <col span="1" style="width: 12%;">
-              <col span="1" style="width: 12%;">
-              <col span="1" style="width: 60%;">
+              <col span="1">
+              <col span="1">
+              <col span="1">
+              <col span="1">
+              <col span="1" style="width: 40%;">
               <col span="1" style="width: 4%;">
```
Comment (OmerAli277) [inline src/components/sales/AddEditProduct.vue]: Specify percentages

File: src/store/models/user.ts [hunk 2]
```diff
@@ -16,8 +16,14 @@ export interface User extends DatesInterface {
   user_type?: string;
   contact_number?: string;
   credit?: number;
+  user_extra?: UserExtra
 }
 
+export interface UserExtra extends DatesInterface {
+  description?: string;
+}
```
Comment (OmerAli277) [inline src/store/models/user.ts]: Add id field

File: src/components/sales/Order.vue [hunk 3]
```diff
@@ -901,17 +898,24 @@ export default defineComponent({
...
+      }
+
       const user: User = {
         username: this.user.userName,
+        contact_number: this.user.userName,
         first_name: this.user.firstName,
         last_name: this.user.lastName,
-        company:companyId
+        company:companyId,
+        user_extra: user_extra
```
Comment (OmerAli277) [inline src/components/sales/Order.vue:911]: Call user_extra API. Do not change user api.

**Review by OmerAli277 — CHANGES_REQUESTED** [review]

File: src/components/sales/AddEditProduct.vue (hunk 1 above)
Comment (ammarhaiderak) [reply src/components/sales/AddEditProduct.vue]: done

File: src/store/models/user.ts (hunk 2 above)
Comment (ammarhaiderak) [reply src/store/models/user.ts]: done

**Review by OmerAli277 — APPROVED** [review]

**Pull request merged**